packages/react-events/src/Press.js

```diff
@@ -148,9 +154,14 @@ function createPressEvent(
   let pageY = null;
   let screenX = null;
   let screenY = null;
+  let altKey = false;
+  let ctrlKey = false;
+  let metaKey = false;
+  let shiftKey = false;
 
   if (event) {
     const nativeEvent = (event.nativeEvent: any);
+    ({altKey, ctrlKey, metaKey, shiftKey} = nativeEvent);
```

Comment:
why the extra `()` here? prettier?

Reply:
You need it, otherwise it doesn't assign to those bindings. Prettier will add them in anyway.